extensions/prototype-content-with-issues-table/app.R

```diff
@@ -4,10 +4,11 @@ library(DT)
 library(connectapi)
 library(dplyr)
 library(purrr)
+library(tidyr)
 
 # cache data to disk with a refresh every 8h
 shinyOptions(
-  cache = cachem::cache_disk("./app_cache/cache/", max_age = 60 * 60 * 8)
+  cache = cachem::cache_disk("./app_cache/cache/", max_age = 60 * 60 * 12)
```

Comment:
Thought while I'm waiting for the cache to rebuild so I can poke at this: if this takes upwards of an hour to run on some servers, maybe we should have the cache be longer than twelve hours so we reduce the likelihood of someone hitting it? Or, we could consider a different caching mechanism (or maybe shiny caching can do this and I / we just don't know about it?) that lets the cache be refreshed in the background so that there's always _something_ available to look at even if it's not the freshest.

Reply:
I am not sure caching is working as we expect on Connect. Perhaps with min procs set to 1 the cache would rebuild every 12 hours, but as it is now it seems to rebuild when the app is visited and the cache is older than the provided `max_age`.

Reply:
Another idea that might work if the shiny caching isn't doing what we're hoping for: could we store this data as a pin and then update it out of band? I'm not saying that is the only (or even the best way), just a thought that might be helpful here

Reply:
I just came back to poke at the deployed version at https://connect.posit.it/connect/#/apps/bc3dc19c-6c49-4d8b-9db0-c2e629a050b9/ and I'm still getting the error. I assume you're still working on this, but if not just FYI it's still happening. I'm signing off in a bit, but let me know once this is back to ready and I'll take a look (though it might be next week when I get back, sorry!)

Reply:
Is this up + testable now?

Reply:
This was redeployed following your comment on Friday, but we are still subject to the cache expiring and rebuilding when the app is run again (this happened just now, data should populate in ~1h). 

I set minimum processes to `1` on the connect.posit.it deploy of this content to allow for rebuilds without the content being visited. Dogfood refreshes the cache significantly faster and I have yet to see the `libcurl` error when pulling in usage data on that server, so min procs on the Dogfood deploy were left as-is.

Reply:
I think we can continue to have `bindCache()` generate the RDS caches of the content list, usage, and failed jobs, and start using `pins` to create a board from the failed jobs RDS file prior to checking if the cache expired. We can then use the pinned board to generate the failed jobs table.

Reply:
[Leaving my in-code comment here as well](https://github.com/posit-dev/connect-extensions/pull/39#pullrequestreview-2700110285) since it helps address the cache refresh time issues noted above:

> As discussed in the Metrics sync today we needed to introduce temporary date range restrictions to improve load times on connect.posit.it. Until we surface failed content job data via v1/content/failed we are filtering to content deployed within the last 60 days. Additionally, since completed jobs are reaped every 30 days by default we only collect usage data for visits within that last 30 days.